inst/benchmarking.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Benchmarking mlr3fda"
+output: html_document
+---
+
+```{r}
+library(data.table)
+library(mlr3fda)
+library(mlr3pipelines)
+
+setDTthreads(threads = 1)
+
+generate_data <- function(n_patients = 100, n_weeks = 10, seed = 1234) {
```

Comment:
I think it makes sense to benchmark both the irreg and the reg case (if we optimize the reg case)

Reply:
but we can basically also just use the same data and then convert to `tfd_irreg` and `tfd_reg`